crates/matrix-sdk-ffi/Cargo.toml

```diff
@@ -24,6 +24,7 @@ futures-util = { version = "0.3.17", default-features = false }
 matrix-sdk = { path = "../matrix-sdk", features = ["experimental-timeline", "markdown"] }
 once_cell = "1.10.0"
 parking_lot = "0.12.0"
+ruma = "0.6.1"
```

Comment:
Not sure if Ruma was being intentionally avoided as a dependency on the FFI, so might need to rethink things a bit?

Reply:
You can use it as `matrix_sdk::ruma`, but I'm not sure it was intentionally avoided.